packages/ketcher-core/src/application/render/restruct/reatom.ts

```diff
@@ -1109,6 +1095,36 @@ function pathAndRBoxTranslate(path, rbb, x, y) {
 }
 
 function bisectLargestSector(atom: ReAtom, struct: Struct) {
+  const { largestAngle, neighborAngle } = getLargestSectorFromNeighbors(
+    atom,
+    struct
+  )
+  const bisectAngle = neighborAngle + largestAngle / 2
+  return newVectorFromAngle(bisectAngle)
+}
+
+function trisectionLargestSector(atom: ReAtom, struct: Struct) {
+  const { largestAngle, neighborAngle } = getLargestSectorFromNeighbors(
+    atom,
+    struct
+  )
+  const firstTrisectorAngle = neighborAngle + largestAngle / 3
+  const secondTrisectorAngle = neighborAngle + (largestAngle * 2) / 3
+
+  return [
+    newVectorFromAngle(firstTrisectorAngle),
+    newVectorFromAngle(secondTrisectorAngle)
+  ]
+}
+
+function newVectorFromAngle(angle: number): Vec2 {
+  return new Vec2(Math.cos(angle), Math.sin(angle))
+}
+
+function getLargestSectorFromNeighbors(
+  atom: ReAtom,
+  struct: Struct
+): { neighborAngle; largestAngle } {
```

Comment:
Maybe remove the return type here? This line makes the reture type `{ neighborAngle: any, largestAngle: any}`, but it has been implicitly infered as `{ neighborAngle: number, largestAngle: number}`

Reply:
fixed